driver-testsuite/web-fixtures/advanced_form_post.php

```diff
@@ -13,11 +13,15 @@
 }
 
 $_POST['agreement'] = isset($_POST['agreement']) ? 'on' : 'off';
+foreach ($_POST as $key => $value) {
+	unset($_POST[$key]);
+	$_POST[htmlspecialchars($key, ENT_QUOTES, 'UTF-8')] = htmlspecialchars($value, ENT_QUOTES, 'UTF-8');
```

Comment:
rather than replacing $_POST, I would rather use a separate array if we go this way